### DISCUSSION TITLE
Reset password screen redirects to login screen

### DISCUSSION THREAD

**Discussion by louiswol94** [directus/api#2394] [Q&A]
When requesting a password reset I have the following issues:

1) The Directus logo is missing (https://directus.io/assets/directus-white.png)
2) When clicking on the "Reset my password" link it redirects me to the login screen.

The link in the email seems correct:
https://mydomain.com/admin/#/reset-password?token=eyJ0eXAiOiJK....
But then it just redirects to https://mydomain.com/admin/#/login directly without asking me to fill in a new password.

Any ideas here?

Thanks!

**Comment by louiswol94** [comment]
![Screenshot 2021-02-12 at 12 02 18](https://user-images.githubusercontent.com/20996103/107754526-46465300-6d2a-11eb-95da-4527b217aa2d.png)

**Reply by benhaynes** [reply]
What version of Directus are you using?

**Reply by louiswol94** [reply]
@benhaynes Sorry i should have included this info. V 8.8.1

**Reply by louiswol94** [reply]
@benhaynes Any ideas here? :-)

**Reply by benhaynes** [reply]
Unfortunately, no. My recommendation would be to upgrade to the latest Directus version... our team is too small to monitor/triage legacy versions.

**Reply by louiswol94** [reply]
Understandable. Thanks Ben!
